build.sbt

```diff
@@ -0,0 +1,25 @@
+name := "dfdl-envelope-payload"
+
+organization := "io.github.dfdlschemas"
+
+version := "0.1.0-SNAPSHOT"
+
+scalaVersion := "2.12.18"
+
+retrieveManaged := true
+
+useCoursier := false // required for retrieveManaged to work so that lib_managed gets populated
+
+libraryDependencies ++= Seq(
+  "io.github.dfdlschemas" % "dfdl-tcpmessage" % "0.1.0-SNAPSHOT",
+  "com.owlcyberdefense" % "dfdl-mil-std-2045" % "1.3.2",
+  "com.tresys" % "dfdl-pcap" % "1.3.0",
+  "com.github.sbt" % "junit-interface" % "0.13.3" % "test",
+  "junit" % "junit" % "4.13.2" % "test",
+  "org.apache.daffodil" %% "daffodil-tdml-processor" % "3.6.0" % "test",
+  "org.apache.logging.log4j" % "log4j-core" % "2.20.0" % "test",
```

Comment:
The log4j dependency is no longer needed in newer versions of Daffodil. We've switch to slf4j and have a custom slf4j-logger dependency used by the tdml-processor.